### PR TITLE
Convert middleware to request handler

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️

### CODE REVIEW

File: src/Middleware/SwaggerJson.php [hunk 1]
```diff
@@ -8,13 +8,12 @@
...
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
-use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 use Yiisoft\Cache\CacheInterface;
 use Yiisoft\DataResponse\DataResponseFactoryInterface;
 use Yiisoft\Swagger\Service\SwaggerService;
 
-final class SwaggerJson implements MiddlewareInterface
+final class SwaggerJson implements RequestHandlerInterface
```
Comment (samdark) [inline src/Middleware/SwaggerJson.php:16]: 1. What's the benefit?
2. It's still in the `Middleware` namespace.

Comment (xepozz) [reply src/Middleware/SwaggerJson.php:16]: I'll move it outside of the `middleware` folder

**Review by vjik — CHANGES_REQUESTED** [review]
Need fix readme

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
@xepozz readme should be adjusted. Afterwards, it's good to merge.

**Comment by xepozz** [comment]
Done

**Review by vjik — CHANGES_REQUESTED** [review]
Need add configuration of new actions to `di-web.php`.

**Review by vjik — APPROVED** [review]

**Pull request merged**